sdk/daml-lf/interpreter/src/test/scala/com/daml/lf/speedy/UpgradeTest.scala

```diff
@@ -113,15 +110,37 @@ class UpgradeTest(majorLanguageVersion: LanguageMajorVersion)
     """
   }
 
+  val pkgId4: Ref.PackageId = Ref.PackageId.assertFromString("-pkg4-")
+  private lazy val pkg4 = {
+    // add an optional additional signatory
```

Comment:
This is already done by pkg3. I've stared some time at the screen but I can't tell the difference between pkg3 and pkg4. Are they the same?